src/components/atoms/button/stories/button.stories.native.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react-native';
+import { action } from '@storybook/addon-actions';
+import { Text, View } from 'react-native';
+
+import Button from '../';
+
+storiesOf('Atoms/Button', module)
```

Comment:
Does it seem like there's a fair bit of overlap here with `button.stories.common`?

Reply:
@jh3y those stories (button) just an examples of use different stories for different platforms and how you can reuse same stories for both web and native. Yeah, some of them overlaps but this is fine for purpose of demonstration.